### PR TITLE
Changes to the documentation

### PR DESCRIPTION
**Pull request by melisaldiaz**

### CODE REVIEW

File: blake3/lib/BLAKE3.hs [hunk 1]
```diff
@@ -118,9 +118,9 @@ hasherKeyed key0 = unsafeDupablePerformIO $
 -- | Update 'BIO.Hasher' with new data.
 update
   :: forall bin
-  .  BA.ByteArrayAccess bin
+  .  BA.ByteArrayAccess bin
```
Comment (melisaldiaz) [inline blake3/lib/BLAKE3.hs:121]: Sorry about the white space!

Comment (k0001) [reply blake3/lib/BLAKE3.hs:121]: No worries!

**Comment by k0001** [comment]
Excellent! Thank you! 

Could you mention “Documentation fixes” in the `CHANGELOG.md` file? Thanks.

**Comment by k0001** [comment]
Amazing! Thanks!

**Pull request merged**